crates/boundless-cli/src/config.rs

```diff
@@ -34,6 +34,10 @@ pub struct GlobalConfig {
     #[clap(long, env = "RPC_URL", global = true)]
     pub rpc_url: Option<Url>,
 
+    /// Limit the requests per second to the RPC endpoint
+    #[clap(long, env = "RPS", global = true, default_value_t = 25)]
```

Comment:
Not sure defaulting the value to 25 is a good idea.  I'd say that the majority of provers are not using a free RPC endpoint, so they would be affected.

Also for backwards compatibility, set the default to unlimited (current behaviour), but allow one configure to a different value